src/lib/features/client-feature-toggles/client-feature-toggle.controller.ts

```diff
@@ -69,6 +70,8 @@ export default class FeatureController extends Controller {
 
     private eventBus: EventEmitter;
 
+    private clientFeaturesCacheMap = new Map<string, number>();
```

Comment:
We can not read the size of memoizee directly, so we will be storing the memory usage for each query key into new cache. This will be calculated on new revision id.